src/main/java/io/numaproj/numaflow/sourcer/Offset.java

```diff
@@ -0,0 +1,25 @@
+package io.numaproj.numaflow.sourcer;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Offset is the message offset.
+ */
+@Getter
+@Setter
```

Comment:
Can we use lombok @Data which includes getter, setter and all-construct?

Reply:
because of the java doc I didn't go with Lombok, since it's exposed to users.